packages/synapse-client/src/util/SynapseClientError.ts

```diff
@@ -1,28 +1,31 @@
-/**
- * Error message returned by the Synapse backend joined with the
- * HTTP status code.
- */
 import {
-  ErrorResponse,
+  BaseError,
+  DrsErrorResponse,
+  OAuthErrorResponse,
   TwoFactorAuthErrorResponse,
-} from '@sage-bionetworks/synapse-types'
+} from '../generated'
 
-import { SynapseError } from './SynapseError'
+type SynapseError =
+  | BaseError
+  | DrsErrorResponse
+  | OAuthErrorResponse
+  | TwoFactorAuthErrorResponse
 
+/**
+ * Error message returned by the Synapse backend joined with the
+ * HTTP status code.
+ */
 export class SynapseClientError extends Error {
```

Comment:
Moved this class into `synapse-client` because our `fetch` implementation (`fetchWithExponentialTimeout`) may create an instance of this class.